M/MUMPS/MUMPS_seq@5/build_tarballs.jl

```diff
@@ -91,4 +100,4 @@ dependencies = [
 ]
 
 # Build the tarballs
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat = "1.6")
```

Comment:
Can I just exclude the platform? Or should I set a gcc_version instead?

Reply:
I think doing
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat = "1.6", preferred_gcc_version=v"5")
```
is simpler than excluding a platform which would make Valentin sad :stuck_out_tongue: